From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@YanZhangADS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (zhangya). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
The "authors" field accepts only one author. Please edit the "authors" field. Thanks.
 For guidance, see https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/article-metadata.md

rmca14 commented:
Please resolve the merge conflict. For guidance, see http://sharepoint/sites/azurecontentguidance/wiki/Pages/Resolve%20a%20local%20merge%20conflict%20yourself.aspx

Thank you

rmca14 commented:
Closed due to inactivity.

tdykstra committed:
add C# precompiled

tdykstra committed:
clean up switcher and next steps includes

PRMerger20 committed:
Merge pull request #28562 from lleonard-msft/fix/vm-tutorial-kv-links

Fixing broken links in 'Generate certificate' section.

ktoliver committed:
Merge pull request #28680 from v-nagta/googleappsupdate

USER STORY 1134730 SaaS App Tutorial: Google Apps Tutorial Update

anumjs committed:
Removed .NET code snippet

btardif committed:
fixing grammar for recomendation pt2

cherylmc committed:
update

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into reset

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

cherylmc committed:
fortinet

Commit:
fixed metadata titles and capitalized Git

cherylmc committed:
format

markgalioto committed:
updating the MABS support matrix article

anumjs committed:
Incorporated Acrolinx suggestions

Commit:
formatting and edits to task topic

cherylmc committed:
Update how-to-routefilter-powershell.md

Commit:
4. Remove Firewall bullet from includes file.  Finish all other 'sql-database-connect-query-*.md' files.

PRMerger10 committed:
Merge pull request #28780 from cherylmc/fortinet

update fortinet

Commit:
minor edits

ktoliver committed:
Merge pull request #28768 from btardif/master

updating retention policy for metrics, they are all now 30 days as a …

ktoliver committed:
Merge pull request #28775 from RicksterCDN/FlashGrid

adding support statement

Commit:
fixing index link- again

Commit:
fixing index link- again2

PRMerger19 committed:
Merge pull request #28784 from cherylmc/patch-5

fix typo

Commit:
5. In -go.md, line 30 Firewall bullet, path is now being fixed.

Commit:
Updated images to remove personal information.

Commit:
Added "(VM)" to first line of document so that all other references to VM are understood.

Commit:
fixing index link- again3

ktoliver committed:
Merge pull request #28594 from justcla/PublishFromVS

New doc: Publish Web App from Visual Studio

adegeo committed:
Added new step in tutorial

adegeo committed:
Fixed extro

JasonWHowell committed:
Update howto-configure-server-logs-in-cli.md

PRMerger12 committed:
Merge pull request #28754 from Juliako/patch-1

Update media-services-release-notes.md

PRMerger19 committed:
Merge pull request #28747 from MGoedtel/UpdateLinuxAgent1128

Update linux agent1128

JasonWHowell committed:
Update howto-configure-server-parameters-using-cli.md

spelluru committed:
V1

Commit:
6. Minor visual fixes.

Commit:
Merge pull request #28790 from JasonWHowell/patch-1

MySQL: Update howto-configure-server-logs-in-cli.md

JasonWHowell committed:
Update tutorial-php-mysql-app.md

spelluru committed:
Removed Self-hosted IR as it is not relevant to this quickstart.

JasonWHowell committed:
Update tutorial-design-database-using-cli.md

JasonWHowell committed:
Update quickstart-create-mysql-server-database-using-azure-cli.md

PRMerger13 committed:
Merge pull request #28792 from JasonWHowell/patch-2

MySQL: Update howto-configure-server-parameters-using-cli.md

JasonWHowell committed:
Update howto-restore-server-cli.md

JasonWHowell committed:
Update howto-manage-firewall-using-cli.md

PRMerger13 committed:
Merge pull request #28797 from JasonWHowell/patch-1

MySQL: Update howto-restore-server-cli.md

msjuergent committed:
change headers2

Commit:
Merge pull request #28796 from JasonWHowell/patch-5

MySQL: Update quickstart-create-mysql-server-database-using-azure-cli.md

Commit:
Merge pull request #28794 from JasonWHowell/patch-4

MySQL: Update tutorial-design-database-using-cli.md

PRMerger19 committed:
Merge pull request #28793 from JasonWHowell/patch-3

WebApps: Update tutorial-php-mysql-app.md

JasonWHowell committed:
Update app-service-web-create-resource-group-no-h.md

Commit:
Merge pull request #28798 from JasonWHowell/patch-2

MySql: Update howto-manage-firewall-using-cli.md

JasonWHowell committed:
Update tutorial-docker-python-postgresql-app.md

JasonWHowell committed:
Update tutorial-docker-python-postgresql-app.md

yanancai committed:
fix blocking issues

JasonWHowell committed:
Update tutorial-design-database-using-cli.md

Commit:
Merge pull request #28801 from JasonWHowell/patch-1

WebApps: Update tutorial-docker-python-postgresql-app.md

PRMerger20 committed:
Merge pull request #28802 from JasonWHowell/patch-2

MySQL: Update tutorial-design-database-using-cli.md

JasonWHowell committed:
Update known-issues-and-troubleshooting-guide.md

GitHubber17 committed:
Merge pull request #28710 from anumjs/shardmap

Added Java API links

PRMerger20 committed:
Merge pull request #28803 from JasonWHowell/patch-1

ML: Update known-issues-and-troubleshooting-guide.md

spelluru committed:
Fixed issues

spelluru committed:
Minor changes

GitHubber17 committed:
Merge pull request #28800 from JasonWHowell/patch-4

Update app-service-web-create-resource-group-no-h.md

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into armtutorial1128

GitHubber17 committed:
Merge pull request #28529 from yanancai/master

Azure Data Lake Tools for Visual Studio Export Database

spelluru committed:
Added note about special characters

spelluru committed:
Numbered list -> bulleted list

PRMerger10 committed:
Merge pull request #28805 from spelluru/adfssis1128

Added note about special characters

mimig1 committed:
Cosmos DB: Modified information to account for MongoDB accts

spelluru committed:
adding to the TOC

msjuergent committed:
change headers3

Commit:
7. Rephrased outer bullet text, in the first new includes file, to address concern of Beth-Anne.  Fixing blocker about numbering restarting at 1 when it looked like it should continue on to 5.  Confirming the one day increment of dates is intentional.

GitHubber17 committed:
Merge pull request #28806 from mimig1/patch-170

Cosmos DB: Modified information to account for MongoDB accts

GitHubber17 committed:
Merge pull request #28302 from msjuergent/hanaoperations

first change

snehaamicrosoft committed:
pricing, hyper-v related changes

neilpeterson committed:
minor updates

PRMerger16 committed:
Merge pull request #28782 from markgalioto/ab-11282017

updating the MABS support matrix article

PRMerger13 committed:
Merge pull request #28807 from neilpeterson/aks-sp-update

minor updates

MikeDodaro committed:
Edit per review

tamram committed:
updating author for storage MVC

tamram committed:
updating author for storage MVC - part 2

Commit:
Merge pull request #28810 from tamram/tamram-1127

updating author for storage MVC

PRMerger14 committed:
Merge pull request #28813 from tamram/tamram-1128

updating author for storage MVC - part 2

ruoxinCheng committed:
Update eclipse and intelliJ docs

dominicbetts committed:
Clarify route information

PRMerger19 committed:
Merge pull request #28818 from ruoxinCheng/HDI

Update eclipse and intelliJ docs

PRMerger16 committed:
Merge pull request #28817 from dominicbetts/clarify-routes

Clarify route information

rayne-wiselman committed:
Update site-recovery-replicate-azure-to-azure.md

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update azure-to-azure-tutorial-failover-failback.md

rayne-wiselman committed:
Update index.yml

rayne-wiselman committed:
Update site-recovery-replicate-azure-to-azure.md

Commit:
whatsnew05

dominicbetts committed:
Keep in sync with central

Commit:
Merge pull request #28823 from dominicbetts/sync-central

Keep in sync with central

PRMerger17 committed:
Merge pull request #28822 from MarkusVi/whatsnew05

whatsnew05

cephalin committed:
#3779

mrbullwinkle committed:
Resolve blocking issues

carolinacmoravia committed:
M16522: Possible typo mistake: "splitproofed"

Hello, @rnagpal,

Translator has reported possible source content issue. It seems that there is a missing space betweeen those words.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

CarlRabeler committed:
Update sql-database-setup-geodr-and-failover-pool-powershell.md

bandersmsft committed:
Azure policy - fixing typo

Commit:
Merge pull request #28828 from bandersmsft/master

Azure policy - fixing typo

GitHubber17 committed:
Merge pull request #28789 from Thraka/svcfab-tutorial

Add new upgrade cluster tutorial part

rjagiewich committed:
Merge pull request #28757 from MightyPen/daveng-go-connectquery-4sb-20171128-a1

DavEng Go ConnectQuery.

snehaamicrosoft committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into migrate

PRMerger18 committed:
Merge pull request #28490 from tdykstra/cleanup

standardize binding ref doc formatting

MikeDodaro committed:
Edit per review 2

rjagiewich committed:
Merge pull request #28532 from rayne-wiselman/rayne-november27-2

adding conceptual TOC for migrate

adegeo committed:
fix some bugs

tdykstra committed:
add webhook notes

GitHubber17 committed:
Merge pull request #28824 from cephalin/public

#3779

itechedit committed:
Merge pull request #28720 from spelluru/armtutorial1128

Quickstart ARM

Commit:
Merge pull request #28829 from snehaamicrosoft/migrate

Migrate

mrbullwinkle committed:
Address H1 intro concern

MikeDodaro committed:
omit needless words

GitHubber17 committed:
Merge pull request #28160 from tdykstra/bindings

add 2.x to bindings table

PRMerger14 committed:
Merge pull request #28827 from CarlRabeler/patch-367

Update sql-database-setup-geodr-and-failover-pool-powershell.md

tdykstra committed:
add non-return-value attribute comment

sethmanheim committed:
Fix typo per customer

Commit:
toc update

GitHubber17 committed:
Merge pull request #27979 from mrbullwinkle/mrb11152017

draft 001

tdykstra committed:
acrolinx

PRMerger13 committed:
Merge pull request #28833 from sethmanheim/eh-fix

Fix typo per customer

Commit:
prebuilt domain reference

PRMerger14 committed:
Merge pull request #26969 from AjayKumar-N/patch-7

(ForumsTeam) the same documentation has been referenced two times.

PRMerger13 committed:
Merge pull request #28826 from carolinacmoravia/patch-492

M16522: Possible typo mistake: "splitproofed"

PRMerger14 committed:
Merge pull request #28830 from Thraka/svcfab-tutorial

fix some bugs

Ja-Dunn committed:
Merge pull request #28206 from ShawnJackson/stream-analytics-high-frequency-trading

edit pass: stream-analytics-high-frequency-trading

GitHubber17 committed:
Merge pull request #28715 from tdykstra/reassign

reassign author metadata

mtillman committed:
Fixed links

rjagiewich committed:
Merge pull request #28732 from MikeDodaro/endpoints-summary-1

Endpoints summary 1

davidmu1 committed:
removed note

mtillman committed:
fixed typos

Commit:
giving up on index link - swapping to what is AML

yujhongmicrosoft committed:
Update documentation-government-functions.md

PRMerger20 committed:
Merge pull request #28375 from carolinacmoravia/patch-467

C6755: LOCBLOCKER: Missing spaces that are blocking content for loca…

v-shils committed:
Merge pull request #26802 from tdykstra/cscpt3

add precompiled c# to Notification Hubs

msmbaldwin committed:
Revised PCI article

Commit:
Merge pull request #28837 from davidmu1/appgatewaytemplate

removed note

PRMerger13 committed:
Merge pull request #28840 from yujhongmicrosoft/patch-7

Update documentation-government-functions.md

Commit:
fixing index link- again4

rjagiewich committed:
Merge pull request #27165 from tdykstra/cscpt4

add precompiled C# info to http

Commit:
task 634: remove public directive from CacheControl examples, update TOC

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

bandersmsft committed:
cost-mgt adding Use reports

mtillman committed:
Fixed first heading to make H1

v-shils committed:
Merge pull request #28260 from tdykstra/cosmos

add precompiled C# to cosmos db binding ref

msmbaldwin committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into blueprints

msmbaldwin committed:
Restored bulletpoints

msmbaldwin committed:
Restored bulletpoints

carolinacmoravia committed:
C8294: Extra content in localiaed versions due to a missing breakline 

Hello, @mumian,

Localization team has reported source content issue that causes localized version to have different format compared to en-us version.  
There is a missing breakline that should be added. 

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

bandersmsft committed:
acrolinx fixes

v-alje committed:
Merge pull request #28847 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #28371 from pkcsf/azure-doc-pkc

Azure doc pkc

MauricioMolinaMicrosoft committed:
Merge branch 'master' into patch-2

rjagiewich committed:
Merge pull request #28287 from tdykstra/twilio

add precompiled C# info to Twilio binding ref

Commit:
Merge pull request #28848 from carolinacmoravia/patch-493

C8294: Extra content in localized versions due to a missing breakline

v-shils committed:
Merge pull request #28355 from tdykstra/mobile

add precompiled C# info to mobile app binding ref doc

Commit:
Merge pull request #28835 from mtillman/patch-2

Fixed links

rjagiewich committed:
Merge pull request #28387 from bradsev/data-science-template

agile template doc

PRMerger12 committed:
Merge pull request #28843 from msmbaldwin/blueprints

Revised PCI article

mimig1 committed:
Cosmos DB: Removed link.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

RicksterCDN committed:
fixed typo

Commit:
controls02

Commit:
Merge pull request #28760 from KumudD/TMFAQ1128

updated Traffic Manager FAQ

nitinme committed:
Adding a release notes entry to the TOC

PRMerger14 committed:
Merge pull request #28854 from MarkusVi/controls02

controls02

hirokib committed:
Update sql-data-warehouse-tables-data-types.md

Commit:
How to: search for an address

cherylmc committed:
Update reset-circuit.md

PRMerger14 committed:
Merge pull request #28853 from RicksterCDN/FlashGrid

fixed typo

Lisaco88 committed:
Merge pull request #28609 from ajlam/master

 Remove db per tenant guidance

KumudD committed:
removed reference to classic portal

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into classicupdateDNS1129

MicrosoftGuyJFlo committed:
Updates for Trusted IPs and Classic Portal

Lisaco88 committed:
Merge pull request #28834 from DeniseMak/11-28-domain-ref-2

GA: Prebuilt domains reference

sanjeev3 committed:
Update try-image-api.md

v-shils committed:
Merge pull request #28845 from dksimpson/dean-634

task 634: remove public directive from CacheControl examples, update TOC

msmbaldwin committed:
Fixed note, bullets

JasonWHowell committed:
Update tutorial-classifying-iris-part-3.md

JasonWHowell committed:
Update tutorial-classifying-iris-part-3.md

Commit:
fix table indent

v-thepet committed:
fix broken links

cherylmc committed:
format

Commit:
Merge pull request #28861 from sanjeev3/master

Update try-image-api.md

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into depmod

cynthn committed:
Update virtual-machines-common-acu.md

Commit:
Merge pull request #28852 from mimig1/patch-171

Cosmos DB: Removed link.

JasonWHowell committed:
Update tutorial-classifying-iris-part-3.md

PRMerger12 committed:
Merge pull request #28865 from v-thepet/newfix

production bug: fix broken links

v-shils committed:
Merge pull request #28846 from bandersmsft/master

cost-mgt adding Use reports

Commit:
Merge pull request #28866 from msmbaldwin/blueprints

Fixed note, bullets

Lisaco88 committed:
Merge pull request #27222 from MauricioMolinaMicrosoft/patch-2

Update Scheduled Events Documentation

Lisaco88 committed:
Merge pull request #28530 from rayne-wiselman/rayne-november27

adding conceptual TOC for A2A

v-shils committed:
Merge pull request #28270 from tdykstra/sendgrid

add C# precompiled to SendGrid binding ref

cherylmc committed:
Update vpn-gateway-howto-point-to-site-classic-azure-portal.md

Commit:
techref02

dsk-2015 committed:
fixed next step in LBS qs

Commit:
Merge pull request #28864 from JasonWHowell/patch-1

ML: Update tutorial-classifying-iris-part-3.md

v-shils committed:
Merge pull request #28777 from cherylmc/reset

new troubleshooting article for ER

cherylmc committed:
Update vpn-gateway-gwsku-include.md

Lisaco88 committed:
Merge pull request #28858 from philmea/lbs-howtosearch

How to: search for an address

Commit:
techref02

PRMerger18 committed:
Merge pull request #28870 from dsk-2015/dsk_iotf17_b2

fixed next step in LBS qs

Lisaco88 committed:
Merge pull request #25657 from JamesEarle/functions-twitter

Updating Images & Instructions for functions-twitter-email sample

v-shils committed:
Merge pull request #28211 from cherylmc/depmod

Depmod

Lisaco88 committed:
Merge pull request #28859 from KumudD/classicupdateDNS1129

Remove classic portal reference

Commit:
Merge pull request #28857 from hirokib/patch-5

Update sql-data-warehouse-tables-data-types.md

carolinacmoravia committed:
P: Broken link in line 39

Hello, @kanshiG,

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.
There is an extra space within link structure.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

ShawnJackson committed:
Update app-service-hybrid-connections.md

PRMerger20 committed:
Merge pull request #28869 from cherylmc/patch-3

Change a note to important - RADIUS and IKEv2 support

Commit:
techref02

cherylmc committed:
Update vpn-gateway-gwsku-include.md

MicrosoftGuyJFlo committed:
Update date on article to reflect change by PG

GitHubber17 committed:
Merge pull request #28574 from tdykstra/cleanup2

clean up switcher and next steps includes

Ja-Dunn committed:
Merge pull request #28613 from HJToland3/master

DMS Quickstart update

GitHubber17 committed:
Merge pull request #28855 from nitinme/nitinme-freshness-updates

Adding release notes to TOC

ShawnJackson committed:
Merge pull request #28196 from Ja-Dunn/app-service-hybrid-connections

edit pass: app-service-hybrid-connections

PRMerger13 committed:
Merge pull request #28872 from cherylmc/patch-4

Update vpn-gateway-gwsku-include.md

mmacy committed:
[aci] logs output update

PRMerger19 committed:
Merge pull request #28871 from MarkusVi/techref02

techref02

cherylmc committed:
Update vpn-gateway-modify-local-network-gateway-cli.md

cherylmc committed:
Update vpn-gateway-modify-ip-prefix-cli-include.md

cherylmc committed:
Update vpn-gateway-modify-ip-prefix-cli-include.md

Commit:
Updated links to Retry Logic examples.  Per 'Purna Chandra Rao Chinta' of Msft Support, and 'Dale Chen' (email from CarlRab 20171129).  MightyPen = genemi.

PRMerger16 committed:
Merge pull request #28887 from mmacy/aci-qs-logs-update

[aci] logs output update

Commit:
Merge pull request #28890 from cherylmc/patch-3

Update vpn-gateway-modify-local-network-gateway-cli.md

PRMerger17 committed:
Merge pull request #28883 from MicrosoftGuyJFlo/MFADate

Update date on article to reflect change by PG

PRMerger18 committed:
Merge pull request #28877 from carolinacmoravia/patch-495

P: Broken link in line 39

Commit:
updates after review cycle

Ja-Dunn committed:
Merge pull request #28891 from cherylmc/patch-4

Update vpn-gateway-modify-ip-prefix-cli-include.md

tfitzmac committed:
consolidate deployment errors

MicrosoftGuyJFlo committed:
Updates for Trusted IPs and Classic Portal

tfitzmac committed:
fix link

CamSoper committed:
emulator -> cluster

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
reword one section

Ja-Dunn committed:
Merge pull request #28860 from MicrosoftGuyJFlo/MFATrustedIPUpdate

Updates for Trusted IPs and Classic Portal

PRMerger18 committed:
Merge pull request #28902 from CamSoper/master

Replacing "emulator" with "cluster"

v-alje committed:
Merge pull request #28903 from MicrosoftDocs/FromPublicRepo

From public repo

GitHubber17 committed:
Merge pull request #28892 from MightyPen/retrysql-xrepo-20171129

Updated links to Retry Logic examples.

PRMerger19 committed:
Merge pull request #28694 from cynthn/l-updates

ls-series updates

Commit:
Merge pull request #28867 from cynthn/master

Update virtual-machines-common-acu.md

Lisaco88 committed:
Merge pull request #28896 from tfitzmac/1129errors

consolidate deployment errors

Commit:
Added deprecation language to CLI 1.0 installation documentation.

PRMerger20 committed:
Merge pull request #28908 from sptramer/deprecate-cli-1.0

Added deprecation language to CLI 1.0 installation documentation.

ktoliver committed:
Merge pull request #28742 from craigcaseyMSFT/v-craic1130

new file including screenshots and updated TOC

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo
